### PR TITLE
B3 Saved and Scheduled Reports

### PR DESCRIPTION
**Pull request by biyeun**
screenshots:
screen shot 2016-02-19 at 5 52 20 pm
screen shot 2016-02-19 at 5 52 29 pm
screen shot 2016-02-19 at 5 52 36 pm
screen shot 2016-02-19 at 5 52 53 pm

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 7 errors:

* corehq/apps/reports/forms.py, line 2 - F401 'MultiInlineField' imported but unused
* corehq/apps/reports/views.py, line 270 - E501 line too long (121 > 115 characters)
* corehq/apps/reports/views.py, line 840 - E501 line too long (125 > 115 characters)
* corehq/apps/reports/views.py, line 849 - E203 whitespace before ','
* corehq/apps/reports/views.py, line 883 - E301 expected 1 blank line, found 0
* corehq/apps/reports/views.py, line 907 - E221 multiple spaces before operator
* corehq/apps/style/forms/widgets.py, line 129 - E201 whitespace after '{'

**Comment by dimagimon** [comment]
There are 7 errors:

* corehq/apps/reports/forms.py, line 2 - F401 'MultiInlineField' imported but unused
* corehq/apps/reports/views.py, line 276 - E501 line too long (121 > 115 characters)
* corehq/apps/reports/views.py, line 846 - E501 line too long (125 > 115 characters)
* corehq/apps/reports/views.py, line 855 - E203 whitespace before ','
* corehq/apps/reports/views.py, line 889 - E301 expected 1 blank line, found 0
* corehq/apps/reports/views.py, line 913 - E221 multiple spaces before operator
* corehq/apps/style/forms/widgets.py, line 129 - E201 whitespace after '{'

**Comment by biyeun** [comment]
on staging and QA ticket submitted

**Comment by millerdev** [comment]
Reviewed pretty quickly.

**Comment by dannyroberts** [comment]
the last few merge commits are because I was working on staging, investigated some conflicts, and then realized `b3-reports-case-form` had just been merged. `b3-reports-case-form+b3-reports-home` is now merged into this branch.

**Comment by biyeun** [comment]
@dannyroberts thanks. I did a rebase for cleaner git history

**Comment by dannyroberts** [comment]
(not mergeable)

**Comment by biyeun** [comment]
@dannyroberts looks like `UITab` refactor caused the conflicts. updated

File: corehq/apps/reports/views.py [hunk 1]
```diff
@@ -805,126 +832,168 @@ def calculate_day(interval, day, day_change):
...
+            if self.is_new:
+                messages.success(request, "Scheduled report added!")
+            else:
+                messages.success(request, "Scheduled report updated!")
+
+            touch_saved_reports_views(request.couch_user, self.domain)
+            return HttpResponseRedirect(reverse('reports_home', args=(self.domain,)))
+
+        return self.get(request, *args, **kwargs)
+
```
Comment (dimagimon) [inline corehq/apps/reports/views.py]: W293 blank line contains whitespace

**Comment by biyeun** [comment]
QA on this passed cc @gcapalbo

File: corehq/apps/reports/templates/reports/reports_home.html [hunk 2]
```diff
@@ -104,61 +121,67 @@
                     </td>
                     <td>{{ report.day_name }} at {{ report.hour }}:00</td>
                     <td>
-                        {# handles old documents #}
+                        handles old documents
```
Comment (gcapalbo) [inline corehq/apps/reports/templates/reports/reports_home.html]: did you mean to uncomment this?

**Comment by gcapalbo** [comment]
just one comment otherwise good on :white_check_mark:

File: corehq/apps/reports/templates/reports/reports_home.html (hunk 2 above)
Comment (biyeun) [inline corehq/apps/reports/templates/reports/reports_home.html]: lol wut. thanks nice catch

**Comment by biyeun** [comment]
@gcapalbo thanks for finding that silly thing. responded, now waiting for ✅

**Comment by gcapalbo** [comment]
:+1: just waiting on build

**Pull request merged**